### PR TITLE
Colorise sample reports

### PR DESCRIPTION
**Pull request by keynmol**
**Published: https://weaver-site.surge.sh/weaver-test/docs/multiple_suites_failures**

Before             |  After
:-------------------------:|:-------------------------:
image |   image

### CODE REVIEW

File: modules/docs/src/main/scala/weaver/Output.scala [hunk 1]
```diff
@@ -35,3 +35,79 @@ object Output {
     } yield value.reduceLeft(_ + "\n" + _)
   }
 }
+
+
+import fansi.Str
+
+object Ansi2Html extends Function1[String, String] {
```
Comment (keynmol) [inline modules/docs/src/main/scala/weaver/Output.scala:42]: This is not a bulletproof implementation - most of it was just to make weaver's examples work.

It probably misses a ton of corner cases.

I can fix it if it ever becomes a problem - ideally I'd want to extract it as a separate mini-project.

File: modules/docs/src/main/scala/weaver/Output.scala [hunk 2]
```diff
@@ -8,21 +8,21 @@ import scala.concurrent.ExecutionContext.Implicits.global
...
 
-  def removeASCIIColors(str: String) =
-    ansiColorRgx.replaceAllIn(str, "")
-
-  def format(s: String) =
-    removeTrailingNewLine(removeASCIIColors(s.replace("repl.SessionApp", "")))
+  def format(s: String) = { 
+    Ansi2Html(removeTrailingNewLine(
+      removeTrailingNewLine(
+        s.replace("repl.MdocSessionApp", "").replace("(none:", "(MyTests.scala:")))
```
Comment (keynmol) [inline modules/docs/src/main/scala/weaver/Output.scala:15]: Fixes the names of suites and the filename `none`

**Comment by keynmol** [comment]
Question: do we want to make more examples contain output rendered as such?

**Comment by Baccata** [comment]
> Question: do we want to make more examples contain output rendered as such?

what would change if the answer was "yes" ?

**Comment by keynmol** [comment]
> What would change if the answer was "yes" ?

Just more outputs, really. For things like https://disneystreaming.github.io/weaver-test/docs/expectations (with assertions) it might be good.

Not a big deal, but we have this ability now so can use it everywhere :D

**Review by Baccata — APPROVED** [review]

**Pull request merged**